### DISCUSSION TITLE
Indexing request rejected by Google Search Console

### DISCUSSION THREAD

**Discussion by nebkoguide** [vercel/next.js#50316] [Help]
### Summary

Every time I try to submit indexing request to Google Search Console, it is giving me this error 👇. Please help me. This is soo frustrating. I'm trying since a week and is giving me same problem.

![image](https://github.com/vercel/next.js/assets/108616642/af1f7077-ffb3-41bf-bca5-213e872fc5a7)

![image](https://github.com/vercel/next.js/assets/108616642/d03fabc4-6556-42e6-8980-b4c16daf8e94)

### Example

https://nebkoguide.com/questions/what-is-the-controlling-idea-or-thesis-of-this-essay-sharing-traditio

**Comment by ciruz** [comment]
Hey @nebkoguide , this has something todo with your Webhost / Server / DNS Settings / Nameserver, not per se with next.js.
When I run a DNS check, everything looks fine. If you host on Vercel, maybe contact their support, depending on your tier, you can create a case or write them on github => https://vercel.com/help#issues

**Comment by bi-zi** [comment]
Same thing for 3 days now
![image](https://github.com/vercel/next.js/assets/76881636/7c7c17e4-22eb-44bf-b854-10cc57028cd0)

**Comment by jaypalsinhjadeja** [comment]


**Reply by ciruz** [reply]
@jaypalsinhjadeja your link opened a anti virus scam website too.

**Reply by jaypalsinhjadeja** [reply]
Sorry for the inconvenience happened, can you please send the screenshot of the website you opened?

**Reply by ciruz** [reply]
Your link opens a scam site, I tested it on desktop. Just disable all ad blockers and ghostery or try it in icognito mode (without any extensions).

![image](https://github.com/user-attachments/assets/43ee4999-5df3-4d9f-865c-424b933ac42f)

This opens this site, even on iOS. 

![image](https://github.com/user-attachments/assets/9a0ec572-a884-4a25-b9d5-5a5ce5ccdcaf)

I would asume its the ad that are embed on the site, if its your website.

**Reply by jaypalsinhjadeja** [reply]
Sorry for all the problems with my website. I have updated my website, now you can visit this link: [Indexing Request Rejected](https://peakblogging.me/indexing-request-rejected/)

**Comment by jaypalsinhjadeja** [comment]
This is a widespread error many bloggers face due to some problems pinned down in the points below:

1. Problem with Robots.txt
2. Technical Errors
3. Incorrect URL
4. Low-Quality Content
5. Duplicated Content
6. Low Domain Authority
7. New Website

But don’t worry we have a full solution to this problem occurring in Google Search Console on the link: [Indexing Request Rejected](https://peakblogging.me/indexing-request-rejected/)
